lib/trento/support/mix/tasks/print_version.ex

```diff
@@ -0,0 +1,10 @@
+defmodule Mix.Tasks.PrintVersion do
```

Comment:
Great! This is the way! I would just rename the command to `mix version` to be consistent whit what we already have generated in the release commands.

Reply:
Renamed in: https://github.com/trento-project/web/pull/545/commits/9e4a9f51c02625470458962f28b6317f80063923